[WCM] Document constraint validator alias made optional

Mx-Glitter opened the pull request:
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no (symfony/symfony#17074) |
| Applies to | 2.8 |
| Fixed tickets |  |

wouterj committed:
Added February changelog

xabbuh committed:
fix changelog syntax

Terminate the `tip` directive with a blank line.

weaverryan committed:
Merge branch '2.7'

* 2.7:
  Added February changelog
  Minor fixes and added Fabric tool
  Fixed case where definition service is an alias
  Fixed grammar issues
  Improved the text of the links
  Updated the contents of the generic Deployment article
  Fixed semantic error

Conflicts:
	changelog.rst

weaverryan committed:
minor #5054 [Changelog] fix changelog syntax (xabbuh)

This PR was merged into the master branch.

Discussion
----------

[Changelog] fix changelog syntax

| Q             | A
| ------------- | ---
| Doc fix?      | yes
| New docs?     | no
| Applies to    | 3.0+
| Fixed tickets |

Terminate the `tip` directive with a blank line.

Commits
-------

653ac94 fix changelog syntax

weaverryan committed:
Merge branch '2.7'

* 2.7: (38 commits)
  Update configuration.rst
  Update configuration.rst
  Readability
  [#4875] Minor language tweaks
  [#4779] Minor tweaks to a huge PR
  [#4724] Minor language tweaks and cross-link to form theming
  [Serializer] Fix CS
  document the validation payload option
  Fixed some of the comments
  [#4793] Very minor tweak that sounds more natural
  [#4732] Tweaking language, clarifying purpose of disabling form and that you can disable CSRF on 1 form
  [#4732] Tweaking language, clarifying purpose of disabling form and that you can disable CSRF on 1 form
  [config][translator] use the new fallbacks option.
  add missing reference options and descriptions
  Linked "logrotate" to its official website
  Minor rewording
  Added a note about log file sizes and the use of logrotate
  Use snake_case for template names
  Fixed minor grammar issues
  Made http cache chapter best-practices-compatible and lots of other fixes
  ...

weaverryan committed:
Merge branch '2.7'

* 2.7: (103 commits)
  Backporting some stuff from 2.7, that I think must have gotten merged only there by accident
  [#5064] Minor language tweaks
  Fixing bad merge conflict (forgot to save!)
  Remove unnecessary component reference
  Correct RegisterListenersPass namespace
  Fix service id
  Switched the first example to a static constructor method
  added some more components for Tobion as a merger
  Fixed variable name in : Reference -> validation constraints -> count -> basic usage -> PHP
  [#5036] Typo fix (probably mine originally) caught by xabbuh
  reword to serves
  Adding a link to define "lts"
  Better wording
  Minor improvement for symfony-installer with LTS
  Updating for new security service names in 2.6
  [#5033] Tweaking variable name to "match" the service id
  [#5017] Minor language tweaks
  [#5015] Updating the security service name for 2.6 - thanks to Cordoval
  [#5015] Very small tweak
  [#5011] Adding one more fix I missed
  ...

weaverryan committed:
Merge branch '2.7'

* 2.7:
  Removing micro-optimization note about @Template

weaverryan committed:
Merge branch '2.7'

* 2.7: (47 commits)
  Removed duplicate "long"s
  terminate file with newline
  move title back and move description to separate file as suggested in PR #4892
  move title out of included file as suggested in PR #4892
  Add possible values for widget_type
  [#4842] Making 2 sentences
  changes as suggested by WouterJ in pull request #4842
  Add meaning of yellow icon for number of queries
  Fixing bad link name
  fix typo in event flow diagrams
  Many fixes thanks to stof, WouterJ, xabbuh and dupuchba
  added Jakub as a merger for the DomCrawler component
  Changes thanks to WouterJ and xabbuh
  [#5094] Removing mkdir - it's not needed (thanks xabbuh)
  some tweaks to #4601
  Moving index down to correct section
  [#4989] Language tweaks and making the example simpler
  Remove useless setLocale() call and add code block with locale setter
  Finaly touches on translation locale setting note
  Review note about setting the translator locale in a controller.
  ...

wouterj committed:
Merge branch '2.7'

wouterj committed:
Merge branch '2.7'

wouterj committed:
Merge branch '2.7'

wouterj committed:
Added March changelog

wouterj committed:
Merge branch '2.7'

weaverryan committed:
Merge branch '2.8'

* 2.8:
  [#5098] Minor fix so controller in dumped apache matches _controller route information above
  fix code block order
  Update process.rst
  [Cookbook] Add warning about Composer dev deps on Heroku
  Added a note against using the Apache Router
  Implemented all the changes suggested by reviewers
  Minor rewording
  Reviewed Configuration cookbook articles
  Reviewed Cache cookbook articles

weaverryan committed:
Merge branch '2.8'

* 2.8:
  Fix broken link in security chapter
  Add version 2.8 to the release roadmap
  bug #5162 Fix misplelled XliffFileLoader class in the Using Domains (Nicola Pietroluongo)
  Fix misplelled XliffFileLoader class in the Using Message Domains example
  Removing a section about Roles that I think has no real use-case
  add missing security advisories
  Fix misplelled XliffFileLoader class in the Using Message Domains example
  Use correct Session namespace

wouterj committed:
Merge branch '2.8'

weaverryan committed:
Merge branch '2.8'

* 2.8:
  Remove horizontal scrollbar
  Fixed typo.
  Minor rewording
  Fixed a minor grammar issue
  Minor rewording
  Fixed some typos
  Improved the explanation about the "secret" configuration parameter

wouterj committed:
Merge branch '2.8'

wouterj committed:
Merge branch '2.8'

wouterj committed:
Merge branch '2.8'

weaverryan committed:
Merge branch '2.8'

* 2.8:
  Rewrite utf8mb4 cautions, add comment into sample configuration
  Add backticks for code-styling
  Indenting caution block to nest it inside the sidebar
  Revert "Fix example name to avoid breaking collision with standard data-collectors"
  Revert "Add a cautionary note telling users where the "standard" data-collector names can be found."
  Add a cautionary note telling users where the "standard" data-collector names can be found.
  Fix example name to avoid breaking collision with standard data-collectors
  Change MySQL UTF-8 examples to use utf8mb4, which is closer to the standard most people would expect
  [Cookbook] Custom compile steps on Heroku
  Added information about the Symfony Demo application
  Renamed precision option to scale

wouterj committed:
Merge branch '2.8'

wouterj committed:
Merge branch '2.8'

weaverryan committed:
Merge branch '2.8'

* 2.8: (25 commits)
  Update doctrine.rst
  Doctrine Custom Mapping
  fixing whitespace
  Update NotBlank to reflect the actual validation
  Updated autoload standard to PSR-4.
  remove unnecessary code
  uppercase "dependency injection"
  use "Symfony Framework" instead of "Symfony framework"
  use "Form component" instead of "form framework"
  use "full-stack" instead of "full stack"
  unify exception usages
  unify event subscriber usages
  use "console command" instead of "Console Command"
  unify EventDispatcher/event dispatcher usages
  use boolean instead of Boolean
  use "object-oriented" instead of "object oriented"
  "web debug toolbar" instead of "Web Debug Toolbar"
  use "stylesheet" instead of "Stylesheet"
  unify "Symfony Standard Edition" usages
  consistency, replace "save-handler" with "save handler"
  ...

wouterj committed:
Merge branch '2.8'

weaverryan committed:
Merge branch '2.8'

* 2.8: (67 commits)
  Update data_collector.rst
  Updated types articles
  Improve invalid user provider exception message
  Check user provider type
  Use configured user provider instead of injection
  Reviewed form type options includes
  Update authentication.rst
  Update authentication.rst
  Remove Serial Comma
  Update validation.rst
  Improve Choice Validation Constraint Example
  Clarify `query_builder` closure return type
  Remove indentations for all-whitespace lines.
  Fix incorrect instructions regarding linking to the "full panel", improve roll-over data example.
  Flesh out twig-template for custom data-collector
  Simply moving new section down a little lower - I believe some of the other options are a bit more common (totally subjective)
  describe the allow_extra_fields form option
  another fix!
  Fixing things thanks to Wouter
  Tweaks to the new using commands in a controller article
  ...

wouterj committed:
Merge branch '2.8'

wouterj committed:
Merge branch '2.8'

weaverryan committed:
Merge branch '2.8'

* 2.8: (21 commits)
  Adding one more note about why we're in config.yml
  [#5302] Re-reading sections after moving them, and tweaking some things that did not make sense anymore
  Update doctrine.rst
  Place DQL in front of QueryBuilder
  Slight re-wording of new paragraph with the goal of being as short as possible
  Fix formatting error
  Created a new section for rotating log files and explained the max_files configuration option
  Fixes after review
  [VarDumper] Add doc for assertDump* assertions
  Changed comment from # to //
  Applied comments
  [BestPractices] restructured text format for the installation instructions template
  Better illustrate what the "user mistake" is.
  Fix typo
  Added new recipe on upgrading a major version
  Fix little title case mistake
  Created 'upgrade' cookbook section
  Added XML and PHP configuration samples
  Added a note about the rotating_file monolog handler
  Changing back to config.yml and fixing some code block mistakes thanks to Wouter
  ...

Commit:
Update create_form_type_extension.rst

weaverryan committed:
Merge branch '2.8'

* 2.8:
  Fix thanks to Wouter
  Added a section about the framework
  Tweaks thanks to Mr @javiereguiluz
  Adding a new entry about deprecation warnings

wouterj committed:
bug #5328 Update create_form_type_extension.rst (jackdelin)

This PR was merged into the master branch.

Discussion
----------

Update create_form_type_extension.rst

Commits
-------

0c089ec Update create_form_type_extension.rst

wouterj committed:
Merge branch '2.8'

wouterj committed:
Merge branch '2.8'

wouterj committed:
Merge branch '2.8'

wouterj committed:
Merge branch '2.8'

norkunas committed:
Remove deprecated parameters

weaverryan committed:
Merge branch '2.8'

* 2.8: (24 commits)
  [#5331] Tiny typo
  [#5373] Small tweak per Stof's comment
  Added depreciation note for the cascade_validation constraint and updated position of depreciation notes
  [PSR-7] Bridge documentation
  Fix after install URL and new photo since AcmeDemoBundle is not part of 2.7
  Fixed a minor grammar issue
  Fixed typos
  Link to the official repository of the bundle.
  Added mentions to some popular (and useful) Symfony bundles
  [#5095] Fixing a typo and updating to a more realistic example
  [Components][ClassLoader] remove DebugClassLoader
  [#4228] Move synthetic services to its own recipe
  clarify bundle installation instructions
  Implemented the suggestions made by Christian and Wouter
  Replace phpDocumentor by the standard PHPDoc
  Implemented the changes suggested by reviewers
  Fixed an internal link reference
  Reviewed the Bundles cookbook articles
  Updated the list of reserved domains and the URL reference
  Use the reserved domains example.com and example.org
  ...

weaverryan committed:
Merge branch '2.8'

* 2.8:
  Remoing link to now-gone web section
  Fixes thanks to Wouter!
  removing reference that I removed
  removing duplicate reference
  Lots of updates thanks to @javiereguiluz!
  adding missing reference
  Proofread!
  Complete rewrite of the page creation chapter to be much shorter
  WIP
  Fixed some issues reported by Wouter
  Fixed minor issue with the title case
  Added a mention to the Symfony Demo application

wouterj committed:
Merge branch '2.8'

weaverryan committed:
Merge branch '2.8'

* 2.8: (122 commits)
  moving options below basic usage
  changing a few orders of sections - mostly to move overridden after the main options
  Rewrite note a bit
  fix capitalization
  tweak headline to be consistent with 2.6
  [Contributing] [Standards] Added entry for identical comparison
  Wrap the table creation inside the class extending Command, so users know where the  comes. They can use it as standalone when needed
  [Serializer] Array Denormalization
  [Cookbook][Assetic] complete a sentence
  Fixing "Undefined method" error on Symfony 2.7
  [#5293] Tweak to link to the article about parent bundles
  Minor tweak to remove FOSUserBundle example and replace with a link to the doc about this
  overriding 3rd party bundles
  Adjust line wrapping
  Add formatting for file and method names
  Update load_balancer_reverse_proxy.rst
  removing upgrading.rst - this has already been moved into several documents in its own section
  [#5166] Minor tweaks
  Revert "Changed dump() to var_dump()"
  Changed dump() to var_dump()
  ...

weaverryan committed:
Merge branch '2.8'

* 2.8: (22 commits)
  Changed PhpStormOpener to PhpStormProtocol
  [#5402] Being explicit what this applies to (it should not apply to things like >=)
  [Contributing] [Conventions] Added entry for Yoda conditions
  Added the "payload" option back
  Show annotations first
  Reordered the code blocks to show Annotations, YAML, XML and PHP
  Fixed the issues reported by @xabbuh
  Finished the documentation of the new data comparison validators
  Added information about the new date handling in the comparison constraints and Range
  Document security.switch_user event
  [#5332] typo
  [#5335] Minor tweaks
  document new Doctrine APC cache service
  Added some more docs about the remember me feature
  [Serializer] Updated the cookbook.
  Fixed a minor grammar issue
  Fixed a minor grammar issue
  [Serializer] ObjectNormalizer, object_to_populate doc. Minor enhancements.
  Added support for standard Forwarded header
  Added support for standard Forwarded header
  ...

weaverryan committed:
Merge branch '2.8'

* 2.8:
  [#5423] Minor tweaks to new voter update
  Added a link to the AbstractVoter cookbook
  Fixed some typos
  Removed the abstract_voter.rst.inc file
  add fixes to abstract_voter include file
  fix problems pointed out by @javiereguiluz and @cordoval
  add & update doc entries on AbstractVoter implementation
  use HTTPS for links to symfony.com
  data transformers cookbook service definition typo
  5370 simplify original sentence
  fixing standard and fixing missing argument in php+xml formats
  Updating for AppBundle and purposefully *not* doing work on configure
  5370 rewrite sentence about fingers crossed handler action level

wouterj committed:
Merge branch '2.8'

weaverryan committed:
Merge branch '2.8'

* 2.8: (74 commits)
  Updated doc references in the map.rst.inc file
  Removed the numbers from the file names
  Made a lot of improvements suggested by reviewers
  removed external references alignement
  made some changes to better integrate the tutorial into the current documentation
  fixed markup
  added the new tutorial in the main index
  move things around
  removed versions when adding Symfony component with Composer
  updated the whole book (changes mainly related to Composer)
  Update part06.rst
  updated code for Symfony 2.3 and made minor tweaks to the text
  reworded slightly some sentence to convert the text from a series of articles to a book
  removed unused references
  added missing links
  removed a note that is not relevant anymore
  added an index file
  removed usage of the ClassLoader component in favor of Composer
  updated composer autoload path
  removed the paragraph about CS as we now have standards
  ...

xabbuh committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

weaverryan committed:
Merge branch '2.7'

* 2.7:
  [#5426] Porting the checkDNS option to 2.7 only
  Reordered the configuration blocks of the first example
  Fixed some errors and made some simplifications
  Added the "payload" option back (was removed by mistake)
  Fixed the order of the examples
  Simplified the first example and added more examples
  [Validator] Updated documentation of URL validator
  Fixed typos
  Removed duplication and moved a caution message
  [Console] Added a cookbook entry on invoking other commands
  [#5367] Making the titles more searchable
  fix #5487
  [FrameworkBundle] Update serializer configuration reference
  Changed Authorization and Authentication to use subsections
  Split Security into Authentication & Authorization

weaverryan committed:
Merge branch '2.8'

* 2.8:
  [#5456] Updating for Symfony 2.7
  Adding an example of how the other format for attaching transformers looks
  fixed typo thanks to @OskarStark
  adding use statement
  using the aliases entity manager service name
  Many tweaks thanks to review
  Completely re-reading the data transformers chapter:

weaverryan committed:
Merge branch '2.8'

* 2.8: (21 commits)
  [#5352] minor language tweak
  Updated text about HTTP methods
  Update http_fundamentals.rst
  [HttpKernel] Fix use statement
  [PSR-7] Fix Diactoros link
  Fixes small typo in data transformers cookbook
  some tweaks to the data transformers chapter
  Updated the Symfony Versions Roadmap image
  Fixed a minor typo
  Fixed minor issues
  Use the built-in serializer instead of promoting JMS
  Removed some information wrongly rebased
  Style / grammar fixes for the deployment chapter
  Style / grammar fixes for the controller chapter
  Style / grammar fixes for the console chapter
  Removes wrong comma
  Rewritten 'whether' & removed serial comma
  Style / grammar fixes for the configuration chapter
  Style / grammar fixes for the cache chapter
  Style / grammar fixes for the bundles chapter
  ...

weaverryan committed:
Merge branch '2.8'

* 2.8:
  Reworded a note about BC
  Added a note about the implementation of the verbosity semantic methods
  Fixed some minor syntax issues.
  Reflect links correctly in gmail.rst
  Revisions to the gmail.rst doc
  Update gmail.rst
  Minor grammar fix.
  Fix RST

weaverryan committed:
Merge branch '2.8'

* 2.8:
  Document Platform.sh on the contributing section.
  Mention Platform.sh on README.
  Better Sphinx version.
  Improve configuration files.
  Prepare Platform.sh configuration files.

weaverryan committed:
Merge branch '2.8'

* 2.8:
  [#5444] Fixing missing public: false declarations and proofing
  Fix RST
  [Book][Routing] Change example to match multiple methods
  Custom voter example, fix missing curly brace
  TYPO: missing closing parantheses of the array
  Fixed an error in the auto_alias format value
  Fixed RST table syntax
  Reverted an unneeded change
  Minor grammar issue
  Minor fixes
  Remove the old voter article
  Rewrite new section
  Move access decision strategy section
  Fixed some errors and added a new note
  Removed an extra blank line
  Added the "versionadded: 2.7" directive
  Documented the "auto_alias" feature
  Minor reword
  Improved the explanation about the verbosity levels of the console

weaverryan committed:
Merge branch '2.8'

* 2.8:
  [#5472] Minor tweak and adding code example
  [#5453] Minor tweaks - mostly thanks to Javier
  Reword
  Fix moving the mapped down as per @xabbuh
  Fix caching
  Fix build
  Fix the rendering issue
  Fix issues reported by @snoek09 . Thank you.
  Added a tip about hashing the result of nextBytes()
  Make the necessary changes mentioned by @xabbuh and add example for form
  rework the quick tour's big picture
  fix for Symfony 2.7
  Fix after install URL and new photo since AcmeDemoBundle is not part of 2.7
  Starting with range documentation
  Improve travis build speed

xabbuh committed:
Merge branch '2.8'

wouterj committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

wouterj committed:
Merge branch '2.8'

wouterj committed:
Merge branch '2.8'

wouterj committed:
Merge branch '2.8'

wouterj committed:
feature #5374 Remove deprecated parameters (norkunas)

This PR was merged into the master branch.

Discussion
----------

Remove deprecated parameters

| Q             | A
| ------------- | ---
| Doc fix?      | yes
| New docs?     | no
| Applies to    | master
| Fixed tickets | -

Commits
-------

d01d7ab Remove deprecated parameters

wouterj committed:
Remove mentions of Symfony1

weaverryan committed:
Merge branch '2.8'

* 2.8:
  Ensure that the entity is updated.
  Update lazy_services.rst
  Update templating.rst
  Additional little check to show how we're assumign the User object is the User entity
  Moved mongodb article and cleanup
  Fixing remaining reference of old emplacement of pdo_session_storage page due to the rebase
  Move some articles from wrong sections, updated indexs
  Move some articles from wrong sections
  [#5374] Add deprecation notice in 2.7
  fix YAML syntax highlighting
  Improve and simplify the contributing instructions about tests
  Some fixes for bundle best practices
  Fix Major upgrade article for 2.7.1 changes
  [#5413] Backporting changes to 2.3
  Added caution notes about the deprecation of container scopes

wouterj committed:
Merge branch '2.8'

wouterj committed:
Added July changelog

wouterj committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

weaverryan committed:
Merge branch '2.8'

* 2.8:
  Reset some more pygments styles
  Use pip instead of submodules
  Use numbered code block
  Add demo warning
  Move Sphinx files to _theme
  Update config
  Use a more recent version of Sphinx
  Temporary disable codeblock
  Use symfony.com theme on Platform.sh builds
  Update page_creation.rst
  [#5593] Very little rewording and improving config
  Updated the profiler matchers article
  don't override existing variables
  [Cookbook][Session] fix default expiry field name
  Fix code

wouterj committed:
minor #5579 [3.0] Remove mentions of Symfony1 (WouterJ)

This PR was merged into the master branch.

Discussion
----------

[3.0] Remove mentions of Symfony1

I think with Symfony 3, we should no longer document how Symfony >1 differs from Symfony 1.

| Q | A
| --- | ---
| Doc fix? | no
| New docs? | no
| Applies to | 3.0+
| Fixed tickets | -

Commits
-------

feb2ab1 Remove mentions of Symfony1

wouterj committed:
Merge branch '2.8'

weaverryan committed:
Merge branch '2.8'

* 2.8:
  [#5480] minor tweaks
  Added instructions about testing a PR
  Integrated PR comments
  Wording
  Fixed numbering
  Fixed case
  Improved sample comment
  Added link to PRs in need of review
  Added initial draft of "Community Reviews" page

weaverryan committed:
[#5480] Adding missing map entry (added 2 so it's easier to find)

weaverryan committed:
Merge branch '2.8'

* 2.8:
  [#5480] Adding missing map entry (added 2 so it's easier to find)

wouterj committed:
Merge branch '2.8'

wouterj committed:
Added August changelog

weaverryan committed:
Merge branch '2.8'

* 2.8:
  Added example on PR IDs
  Simplified/improved wording

wouterj committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

wouterj committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

weaverryan committed:
Merge remote-tracking branch 'origin/master'

* origin/master: (93 commits)
  [#5623] don't configure default constraint message
  [Validator] added BIC validator
  Update links
  Added September changelog
  Added September changelog
  Misspelling
  [#5689] Some minor syntax fixes
  [DI] Add some documentation for the deprecation feature
  Info about implicit session start
  Revert "[#5771] remove another Response constant"
  Revert "Remove not existing response constant"
  [#5771] remove another Response constant
  [#5761] add versionadded directive
  Fixed two typos
  Remember me, Renamed key to secret
  translations have been removed from symfony.com
  Remove not existing response constant
  Update HttpFoundation note after recent changes in routing component
  [#5584] language tweak and XML config fix
  Add DebugBundle config reference
  ...

weaverryan committed:
Merge branch '2.8'

* 2.8: (29 commits)
  Removing getName() as it's not needed in 2.8
  tweaks for 2.7
  Fixing build error
  Tweaks thanks to everyone!
  [WIP] Reworking most of the registration form:
  Fixing build problems and wrong links
  Fixing build error
  Rename CollectionType entry options
  changing includes to links
  Updating book examples to not use deprecated validation methods
  Updated constraint reference docs for Null->IsNull, False->IsFalse, True->IsTrue
  Creating placeholder constraint rst docs for deprecated usage
  Renaming constraint rst files to Is* to preserve edit history
  Fixes thanks to Stof
  fixing build failure
  Tweaks - see #5314
  Minor tweaks
  Final changes
  Reworded the introduction and other minor fixes
  Added a note about the advantages/drawbacks of listeners/subscribers
  ...

weaverryan committed:
Merge branch '2.8'

* 2.8:
  Adding information about empty files sent using BinaryFileResponse.

xabbuh committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

xabbuh committed:
Merge branch '2.8'

wouterj committed:
app/console -> bin/console

wouterj committed:
app/phpunit.xml.dist -> phpunit.xml.dist

wouterj committed:
app/(cache|logs) -> var/(cache|logs)

wouterj committed:
app/bootstrap.php.cache -> var/bootstrap.php.cache

wouterj committed:
Updating 'Exploring the Project'

wouterj committed:
Prefix commands with 'php'

wouterj committed:
[BestPractices] Use new directory structure

wouterj committed:
[QuickTour] Use new directory structure

wouterj committed:
app/console -> bin/console

wouterj committed:
app/phpunit.xml.dist -> phpunit.xml.dist

wouterj committed:
app/cache -> var/cache

wouterj committed:
app/logs -> var/logs

wouterj committed:
app/bootstrap.php.cache -> var/bootstrap.php.cache

wouterj committed:
app/SymfonyRequirements.php -> bin/SymfonyRequirements.php

wouterj committed:
Testing changes

wouterj committed:
Updated directory structures

wouterj committed:
Applied comments

wouterj committed:
Applied suggestions

wouterj committed:
Removed scopes article

weaverryan committed:
Merge branch '2.8'

* 2.8: (35 commits)
  [#5922] Tweaks thanks to reviewers
  Many tweaks thanks to a great review
  language tweak
  several other tweaks
  Completely updating the form type reference section for the text -> TextType changes
  Use path() and url() PHP templating helpers
  Update voter section of best practices
  Added minimal cookbook article about shared
  Document deprecation of supports{Attribute,Class}() methods
  Document Security key to secret renamings
  Use new Simple{Form,Pre}AuthenticatorInterface namespaces
  tweaks thanks to Javier
  adding a section about calling isGranted() from within a voter
  Reworking the voter article for the new Voter class
  fixing build error
  Another huge batch of changes for the form type changes in 2.8
  minor tweaks #5834
  Fixing bad merge
  removing link to an old, non-existent article
  Many fixes thanks to great review from ogizanagi, javiereguiluz and others
  ...

weaverryan committed:
removing more scope stuff

weaverryan committed:
Merge branch '2.8'

* 2.8:
  Finding more old form references
  Remove note about request service, which is not used anymore
  Remove information about request scpoe

weaverryan committed:
removing scope reference

weaverryan committed:
Merge branch '2.8'

* 2.8:
  tweaks thanks to the guys
  removing deprecation note on a section that was removed
  Wrap all strings containing @ in quotes in Yaml
  Updating some places to use the new CustomUserMessageAuthenticationException
  Added a note about the use of _format query parameter
  Always use "main" as the default firewall name (to match Symfony Standard Edition)

weaverryan committed:
feature #5913 [3.0][Book] Use the 3.0 directory structure (WouterJ)

This PR was merged into the master branch.

Discussion
----------

[3.0][Book] Use the 3.0 directory structure

| Q | A
| --- | ---
| Doc fix? | yes
| New docs? | yes
| Applies to | 3.0+
| Fixed tickets | part of https://github.com/symfony/symfony-docs/issues/5898

Commits
-------

25ae8c1 Applied suggestions
ed43015 Prefix commands with 'php'
0d69414 Updating 'Exploring the Project'
6b5c977 app/bootstrap.php.cache -> var/bootstrap.php.cache
3318f2e app/(cache|logs) -> var/(cache|logs)
1461bdc app/phpunit.xml.dist -> phpunit.xml.dist
8794a4d app/console -> bin/console

weaverryan committed:
feature #5916 [3.0][Best Practices][Quick Tour] Use the 3.0 directory structure (WouterJ)

This PR was merged into the master branch.

Discussion
----------

[3.0][Best Practices][Quick Tour] Use the 3.0 directory structure

| Q | A
| --- | ---
| Doc fix? | yes
| New docs? | yes
| Applies to | 3.0+
| Fixed tickets | part of https://github.com/symfony/symfony-docs/issues/5898

Commits
-------

c6d436b [QuickTour] Use new directory structure
215c36d [BestPractices] Use new directory structure

weaverryan committed:
feature #5917 [3.0][Cookbook] Use the 3.0 directory structure (WouterJ)

This PR was merged into the master branch.

Discussion
----------

[3.0][Cookbook] Use the 3.0 directory structure

| Q | A
| --- | ---
| Doc fix? | yes
| New docs? | yes
| Applies to | 3.0+
| Fixed tickets | part of https://github.com/symfony/symfony-docs/issues/5898

Commits
-------

47e11f8 Applied comments
f2be12a Updated directory structures
df20095 Testing changes
ef613f6 app/SymfonyRequirements.php -> bin/SymfonyRequirements.php
af7052b app/bootstrap.php.cache -> var/bootstrap.php.cache
6614c0f app/logs -> var/logs
b3da3b7 app/cache -> var/cache
b6d93f0 app/phpunit.xml.dist -> phpunit.xml.dist
bea4a0c app/console -> bin/console

weaverryan committed:
[#5917] Adding bin

weaverryan committed:
[#5917] Finding some more places with the old directory structure referenced

ogizanagi committed:
Update docs accordingly to min PHP version requirement

Commit:
Document constraint validator alias optional

wouterj committed:
feature #5948 Update 3.0 docs accordingly to min PHP version requirement (ogizanagi)

This PR was merged into the 3.0 branch.

Discussion
----------

Update 3.0 docs accordingly to min PHP version requirement

| Q             | A
| ------------- | ---
| Doc fix?      | yes
| New docs?     | no
| Applies to    | 3.0+
| Fixed tickets | #5947

I'm not sure about the politic regarding the removal of anything mentioning earlier version of PHP for the 3.0 docs ? Should it be reverted ?
Regarding this PR: #5585, it's seems coherent to me to do the same for PHP versions.

Commits
-------

d577848 Update docs accordingly to min PHP version requirement

wouterj committed:
[#5948] Very minor detail

WouterJ commented:
For your information, I added `[WCM]` to your PR title meaning "Waiting Code Merge", as this PR documents on a feature that hasn't made it in Symfony yet.

xabbuh committed:
Merge branch '2.8' into 3.0

Conflicts:
	book/page_creation.rst
	cookbook/deployment/platformsh.rst
	cookbook/symfony1.rst

xabbuh committed:
Merge branch '2.8' into 3.0

xabbuh committed:
[#5956] change app/console back to bin/console

xabbuh committed:
Merge branch '2.8' into 3.0

xabbuh committed:
[#5957] remove deep parameter documentation

wouterj committed:
Merge branch '2.8' into 3.0

Conflicts:
	cookbook/service_container/scopes.rst

wouterj committed:
Added october and november changelog

wouterj committed:
Merge branch '2.8' into 3.0

weaverryan committed:
Merge branch '2.8' into 3.0

* 2.8:
  Minor tweaks
  #5999 fix files names
  Updated the other image too
  Fixed the image
  Minor typo
  Rewords
  Minor changes
  Updated the release process for Symfony 3.x and future releases
  removing php opening tags

xabbuh committed:
Merge branch '2.8' into 3.0

Conflicts:
	best_practices/creating-the-project.rst

zsturgess committed:
Fix #6037

xabbuh committed:
minor #6038 Fix #6037 (zsturgess)

This PR was merged into the 3.0 branch.

Discussion
----------

Fix #6037

| Q             | A
| ------------- | ---
| Doc fix?      | yes
| New docs?     | no
| Applies to    | >= 3.0
| Fixed tickets | #6037

Removed an outdated caution block and rewrote an outdated paragraph.
Hopefully I've done everything correctly *crosses fingers*

Commits
-------

a7effb8 Fix #6037

wouterj committed:
Merge branch '2.8' into 3.0

Conflicts:
	book/service_container.rst

wouterj committed:
Merge branch '2.8' into 3.0

wouterj committed:
Update Symfony version to 3.0

wouterj committed:
Update Symfony version to 3.1

Triiistan commented:
Closing in favor of #6055 on master branch
